README.md

```diff
@@ -149,20 +148,24 @@ More about developing frontend [can be found here](frontend/README.md).
 
 ## Cypress tests
 
-Some [integration tests](https://github.com/Altinn/altinn-studio/tree/main/frontend/testing/cypress) for studio have
-been created with Cypress. More about these tests [can be found here](frontend/testing/cypress/README.md).
+Some [integration tests][17] for studio have been created with Cypress. More about these tests can be found [here](frontend/testing/cypress/README.md)
```

Comment:
Is Cypress still relevant in our integration tests, or can we remove this text with links? I thought we didn't use it anymore.

Reply:
I've asked David, and we concluded with the following: Cypress still gets used, therefore we will include both Cypress and Playwright for now and remove Cypress in the future when it's no longer in use.

Reply:
Understood :)